package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.2",
   "description": "UI for Cognitive Training",
   "author": "ibm-cloud-solutions",
+	"main": "app.js",
```

Comment:
1.  The convention for Hubot scripts is `"main": "index.js"`. I still need to run it, but I expect this will generate some warning messages.
2.  Correct indentation. (Atom doesn't help much here, I use the GitHub editor for this cases.)